sites/svelte.dev/src/routes/+page.svelte

```diff
@@ -6,7 +6,10 @@
 	import Supporters from './_components/Supporters/index.svelte';
 	import Try from './_components/Try.svelte';
 	import WhosUsingSvelte from './_components/WhosUsingSvelte/index.svelte';
+
+	// @ts-ignore
```

Comment:
we should probably have `defaultDirectives` setup for `imagetools` in `vite.config.js`. i.e. change it to:
```
		imagetools({
			defaultDirectives: () => new URLSearchParams('format=avif;webp;png&as=picture');
		})
```

if we included the widths as well then it'd get rid of the query parameters altogether so that you wouldn't need the `@ts-ignore`. though I'm not quite sure what it's 3800 for the largest width here and 3840 for the largest width in the hero and if we need them to be different or not

Reply:
Done